### DISCUSSION TITLE
winutil.exe will not run

### DISCUSSION THREAD

**Discussion by BinDenver** [ChrisTitusTech/winutil#3115] [General]
Just built a new machine and used Microwin 24.11.11 to make the Win 11 24H2 iso. Installed ok, then installed drivers. Then I went to run winutil.exe to do the Tweaks. Opened it under Admin rights via right click menu; it would flash a window for a second then close. I tried a method mentioned here to run the .ps1 from the website via Terminal(Admin) with the command: "Set-ExecutionPolicy Unrestricted -Scope Process -Force
irm christitus.com/win | iex" and that worked fine. But I would really like the .exe file to work going forward. What could the issue with a fresh install be?

**Comment by BinDenver** [comment]
Should I post this in the Issues forum? I just assumed I was doing something wrong.

**Comment by BinDenver** [comment]
please help

**Comment by WirelessSwitcher** [comment]
I have the exact same issue. The terminal screen opens for a split second and shows some red text, but closes so quickly I can't read nor print screen it (especially considering I am trying to run it on an isolated PC with only a VNC session).

**Comment by aprettymess** [comment]
same issue. it was working fine before though.

**Comment by ChrisTitusTech** [comment]
This was resolved in the latest update.
